fms2_io/readme.md

```diff
@@ -603,20 +614,19 @@ deallocate(restart_file)
 - **ncchksz:** Sets chunksize (in bytes) argument in netcdf file creation calls. The default is `64*1024`.
 - **netcdf_default_format:** Sets the netcdf file type. The acceptable values are  "64bit", "classic", "netcdf4". This can be overwritten per file if you specify `nc_format` in the open_file call. The default is 64bit.
 - **header_buffer_val:** Sets the netCDF header buffer size(in bytes). The default is 16384 bytes.
+- **deflate_level:** Determines how much to compress the variable. Chosen by an integer of 1 through 9. The higher the number the more compression will take place, but will take longer to write the file. NOTE: the higher the number, the more time it will take to write the file. The default is no compression. This is loseless compression so that every bit of the original data can be recovered.
```

Comment:
The values of 1-9 are NetCDF imposed and may want to refer people to the NetCDF documentation for a full discussion of this and another other passthrough values for NetCDF manipulations.